.circleci/config.yml

```diff
@@ -16,12 +37,20 @@ jobs:
     executor: linuxgo
     steps:
       - checkout
-      - run: go install -ldflags "-X main.Version=${CIRCLE_TAG}" ./...
-      - run: mkdir -v artifacts; cp -v $GOPATH/bin/buildevents artifacts/
+      - go-build:
+          os: linux
+          arch: "386"
+      - go-build:
+          os: linux
+          arch: amd64
+      - go-build:
+          os: darwin
+          arch: amd64
+      - run: mkdir -v artifacts; cp -v $GOPATH/bin/buildevents-* artifacts/
       - persist_to_workspace:
           root: artifacts
           paths:
-            - buildevents
+            - buildevents-*
```

Comment:
unfortunately it looks as though `persist_to_workspace` does not accept wildcards. I work around this elsewhere by using `tar` to bundle all the things into a predictable name.

Reply:
I tested this to confirm and ... this comment is wrong. `persist_to_workspace` _does_ accept globs. so ... carry on!